cmd/flag/proxy_flags.go

```diff
@@ -30,7 +30,7 @@ var (
 	defaultProxyProtocol = "udp"
 
 	// Supported proxy.
-	availableProxy = []string{"udp", "tcp"}
+	availableProxy = []string{"udp", "tcp", "disabled"}
```

Comment:
On L64 can you edit the comment to say the default behaviour of the protocol flag? (currently defaults to UDP)